docs/SPEC.md

```diff
@@ -2144,6 +2144,32 @@ from(bucket: "telegraf/autogen")
 Records are grouped into a single table.  
 The group key of the resulting table is empty.
 
+#### Columns
+
+Columns lists the column labels of input tables.
+For each input table, it outputs a table with the same group key columns, plus a `_value` column containing the labels of the input table's columns.
+Each row in an output table contains the group key value and the label of one column of the input table.
+So, each output table has the same number of rows as the number of columns of the input table.
```

Comment:
This looks great. The `_value` should be a parameter to the function with a default value of `_value.

Reply:
Ok, shouldn't do the same for `keys`?